Packs/FeedMitreAttack/pack_metadata.json

```diff
@@ -16,5 +16,11 @@
         "MITRE",
         "Feed",
         "ATT&CK"
-    ]
+    ],
+    "dependencies": {
```

Comment:
why do you need to add this?

Reply:
I ran the dependencies command to make sure Demisto REST API is not mandatory anymore, should this be removed?

Reply:
yes